roles/etcd/tasks/permission_directory.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Directory | check certificate permissions
+  file:
+    path: "{{ etcd_config_dir }}"
+    group: "{{ etcd_cert_group }}"
+    state: directory
+    owner: etcd
+    mode: "{{ etcd_cert_dir_mode }}"
+    recurse: yes
+  notify:
+  - Directory set var cert
+
+- name: Directory | check data permissions
+  file:
+    path: "{{ etcd_data_dir }}"
+    state: directory
+    owner: etcd
+    group: etcd
+    mode: "{{ etcd_cert_dir_mode }}"
+    recurse: yes
+  notify:
+  - Directory set var data
+
+- name: Directory | Restart etcd if directory permissions changed
+  service:
+    name: etcd
+    state: restarted
+    enabled: yes
+  when: (etcd_cert_changed|default(false)) or (etcd_data_changed|default(false)) or ((etcd_cert_changed|default(false)) and (etcd_data_changed|default(false)))
```

Comment:
This is not working because the handlers doing set_facts are running at the end of the play, so after this `when:`
To work this should be an handler, but doing chown/chmod while etcd is running is just wrong, what if:
1) you do the chown/chmod
2) etcd creates a new file (so owned by root)
3) you restart etcd
4) now etcd can't access some files

You can only change those file while etcd is stopped, so as said in the previous PR, use `ExecStartPre`

Reply:
fix

Reply:
fix

Reply:
refactoring playbooks